libglusterfs/src/graph.c

```diff
@@ -662,8 +663,12 @@ glusterfs_graph_prepare(glusterfs_graph_t *graph, glusterfs_ctx_t *ctx,
     /* XXX: DOB setting */
     gettimeofday(&graph->dob, NULL);
 
-    fill_uuid(graph->graph_uuid, sizeof(graph->graph_uuid), graph->dob);
-
+    ret = fill_uuid(graph->graph_uuid, sizeof(graph->graph_uuid), graph->dob);
+    if ((ret < 0) || (ret >= sizeof(graph->graph_uuid))) {
```

Comment:
Can't we write the check as
```
if (ret)
    gf_msg(...);
    return -1;
```
And, the same for the code below?

Reply:
I just realized that the return value of `snprintf` can be the copied buffer value as well.
Please ignore this comment.